test/cli/cache-keeps-print-options/cache-keeps-print-options.out

```diff
@@ -1,4 +1,4 @@
-class ::<root> < ::Object () @ (... removed core rbi locs ..., https://github.com/sorbet/sorbet/tree/master/bazel-out/host/genfiles/rbi/procs.rbi#L1, test/cli/cache-keeps-print-options/cache-keeps-print-options.rb:1)
+class ::<root> < ::Object () @ (... removed core rbi locs ..., https://github.com/sorbet/sorbet/tree/master/bazel-out/host/bin/rbi/procs.rbi#L1, test/cli/cache-keeps-print-options/cache-keeps-print-options.rb:1)
```

Comment:
Non-blocking suggestion: change the code that handles `censorForSnapshotTests` to hide / shorten the `procs.rbi` RBI file.

Reply:
This is a great idea, I'll make a follow-up PR to address it.